htdocs/societe/list.php

```diff
@@ -322,7 +322,7 @@
 }
 
 if ($search_status=='') $search_status=1; // always display active thirdparty first
-
+if ($search_type_thirdparty=='') $search_type_thirdparty='0'; // use unknown if the thirdparty type is not set
```

Comment:
I don't understand : 
if $search_type_thirdparty is '', then later nothing will be added as search criteria. So it does what we want to.

Reply:
No, it doesn't. For an unknown reason, the default value is "Other", which is then also taken into account in the filter. The default value must be "Unknown" for the filter to work correctly.

Reply:
I tested it a few times with the current develop branch and it only works correctly with this change. Feel free to test it yourself.